### PR TITLE
GB-1536: Remove null constraint for indexes

### PR DESCRIPTION
**Pull request by Miaxos**
Signed-off-by: Anthony Griffon <anthony@griffon.one>

# Description

Please include a summary of the change and which issue is fixed.
Please also include relevant motivation and context.

# Type of change

- [x] 💔 Breaking
- [ ] 🚀 Feature
- [ ] 🐛 Fix
- [ ] 🛠️ Tooling
- [ ] 🔨 Refactoring
- [ ] 🧪 Test
- [ ] 📦 Dependency
- [ ] 📖 Requires documentation update

### CODE REVIEW

File: cli/crates/server/src/bridge/server.rs [hunk 1]
```diff
@@ -42,15 +42,22 @@ async fn mutation_endpoint(
...
-            error
-        })?;
+    let query = payload.iter_variables().fold(template, Query::bind);
+
+    let mut tx = pool.begin().await.map_err(|error| {
+        error!("can't start transaction: {error}");
+        error
+    })?;
+
+    query.execute(&mut tx).await.map_err(|error| {
```
Comment (yoav-lavi) [inline cli/crates/server/src/bridge/server.rs:52]: Would you mind if we go with `transaction` over `tx`?

**Comment by yoav-lavi** [comment]
Looks great, one comment.
Leaving this open until we have a worker version that supports it

**Review by mlafeldt — APPROVED** [review]

**Comment by yoav-lavi** [comment]
@Miaxos there seems to be an issue with the tests here due to the worker needing to be updated, I'll look into it

**Comment by yoav-lavi** [comment]
Closing since these changes are included in #47

**Pull request closed without merging**